tests/datadriven/test_variants.py

```diff
@@ -187,15 +186,8 @@ def _verifyVariantsCallSetIds(self, searchVariants, searchsampleIds):
     def testSearchAllVariants(self):
         self._verifyVariantsCallSetIds(None, self.vcfSamples[:1])
 
-    def testSearchCallSetIdsSystematic(self):
```

Comment:
Why remove this whole test case? I think the commend just refers to the ``len(callSetIds) == 0`` bit. The test case should still be good.